spec/ibc-setup.md

```diff
@@ -10,11 +10,17 @@ These are the high-level entry-points that call a series of other commands.
 
 ### init
 
-This is used once to setup a new key and configuration file:
+This is used to setup a new key and configuration file.
```

Comment:
This is good. Maybe add a brief comment on this (and link this section) in the README (after quick start when we show how to set up a custom chain)